src/main/java/longah/util/TransactionList.java

```diff
@@ -71,4 +71,51 @@ public void clear() {
     public ArrayList<Transaction> getTransactions() {
         return transactions;
     }
+
+    /**
+     * Returns a String printout the list of transactions stored in the system.
+     */
+    public String listTransactions() {
+        int index = 1;
+        String outString = "";
+        for (Transaction transaction : transactions) {
+            outString = outString + String.format("%d.\n%s", index, transaction) + "\n";
+            index ++;
+        }
+        return outString;
+    }
+
+    /**
+     * Returns a String printout of the list of transactions which the member name is involved as the transaction owner
+     * @param memberName String representation of the name of person to search for
+     */
+    public String findPayments(String memberName) {
```

Comment:
Rename this to `findTransactions`. `findPayments` will be a different method that outputs the least transaction solutions ideally and should be in `MemberList` instead.

Reply:
Done!